server.ts

```diff
@@ -1,24 +1,18 @@
-var express = require("express");
-var { graphqlHTTP } = require("express-graphql");
-var { buildSchema } = require("graphql");
-var cors = require("cors");
-
-// Construct a schema, using GraphQL schema language
-var schema = buildSchema(`
-  type Query {
-    hello: String
-  }
-`);
```

Comment:
before we build a server from schema,
now we use the server, and build a schema from it